### ISSUE TITLE
Is that away ODBC.k not only load data from MySQL but also push tables to MySQL?

### ISSUE THREAD

**Issue by HKby** [KxSystems/kdb#13]
Just want to know if it is possible that kdb+ ODBC.k can insert data / table to MySQL? thanks

**Comment by mlerner1** [comment]
bump

**Comment by CharlieSkelton-Kx** [comment]
yes, although it is limited to submitting the command as text, for example:
```.odbc.eval[handle;"INSERT INTO mytable(textvalue, start_date, end_date) VALUES('sometextvalue','2018-01-09','2018-09-15');"]```
